src/app/space/create/deployments/components/deployment-card/deployment-card.component.ts

```diff
@@ -41,13 +42,13 @@ export class DeploymentCardComponent implements OnDestroy, OnInit {
   };
 
   public cpuConfig: any = {
-    // Seperate chart IDs must be unique, otherwise only one will appear
-    chartId: 'cpu-chart-' + DeploymentCardComponent.chartIdNum++ + '-'
+    // Seperate charts must have unique IDs, otherwise only one will appear
+    chartId: uniqueId('cpu-chart-') + '-'
```

Comment:
Does the trailing ` + '-'` need to be there?

Reply:
I added it since patternfly tacks on the word "sparklineChart" to the id.

Reply:
Huh. Maybe add a comment to that effect then.